src/components/Dashboard/__snapshots__/Dashboard.story.storyshot

```diff
@@ -1886,7 +1886,7 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT|Dashb
             >
               Last updated: 
                
-              Tue, 22 Oct 2019 05:00:00 GMT
+              Tue, 22 Oct 2019 04:00:00 GMT
```

Comment:
What's up with this? You didn't modify the dashboard. 
@scottdickerson is this something with the way dates/timezones are mocked?

Reply:
Confirming that something's up in `master`, I'm seeing the time zone snapshot change from `05:00:00` to `08:00:00`. Corresponds with the VM being set for a different time zone.

Reply:
@tay1orjones I just pushed a fix to this PR to force the timezone